content/reference/api/engine/sdk/examples.md

```diff
@@ -4,11 +4,7 @@ description: Examples on how to perform a given Docker operation using the Go an
   Python SDKs and the HTTP API using curl.
 keywords: developing, api, sdk, developers, rest, curl, python, go
 aliases:
-- /engine/api/getting-started/
-- /engine/api/client-libraries/
-- /engine/reference/api/remote_api_client_libraries/
-- /reference/api/remote_api_client_libraries/
-- /develop/sdk/examples/
```

Comment:
Same for these; or is this to work around the limits we reached? If so, can we fix that somehow?

Reply:
yes, to prune old redirects. I don't think reinstating these would mean we'd hit the limit again. Rather a precautionary cleanup. Keeping too many old redirects around can get kind of difficult to troubleshoot as URL paths have evolved over time, and sometimes you end up with a valid page route that, at some point in time, had been set to redirect to a different URL. Can't easily test for it all either because redirects are handled differently depending on where it's implemented (aliases, redirects.yml, _releaser).

I don't mind adding wildcard redirects though. Looking at the _releaser code it should be fairly straightforward to add.

Reply:
ok, we are still redirecting some of those old `docker_remote_api_v1.1` routes to deprecated api versions, a wildcard redirect will be messy unless we're willing to redirect a whole subtree of things. So I'll just add these back.